docs/src/tutorials/connect-c8y.md

```diff
@@ -149,6 +149,27 @@ To check that this message has been received by Cumulocity,
 navigate to "Device Management/Devices/All devices/<your device id>/Measurements".
 You should observe a "temperature measurement" graph with the new data point.
 
+## Retrieve the JWT Token
+
+Once the Thin Edge device is connected to the Cumulocity cloud using the certificates, it can receive a token that
+can be used later to authenticate HTTP requests.
+
+To retrieve the token from the Cumulocity cloud follow the below steps.
+
+Subscribe to `c8y/s/dat` topic
+
+```
+$ tedge mqtt sub c8y/s/dat --no-topic
+```
+
+Publish an empty message on `c8y/s/uat` topic
+
+```
+$ tedge mqtt pub c8y/s/uat ''
+```
+
+After a while the token will be published on the subscribed topic `c8y/s/dat` in the below format
```

Comment:
For the doc to be complete, I would add at least a link to the c8y documentation on how to use the token.